apps/src/aichat/views/modelCustomization/PublishNotes.tsx

```diff
@@ -126,6 +128,33 @@ const PublishNotes: React.FunctionComponent = () => {
           className={modelCustomizationStyles.updateButton}
         />
       </div>
+      {havePropertiesChanged && !saveInProgress && !isReadOnly && (
```

Comment:
Do we want these in the setup/retrieval tabs as well? This would appear if we have changes in one of the other tabs but no changes in the model card tab as written, yeah?

Reply:
oooooh, yep. This is only the model card tab.

Reply:
extracting these into their own component so it's easier to share them between tabs